Client/src/Views/ProfileView.tsx

```diff
@@ -0,0 +1,225 @@
+import { Box, Button, Collapse, Grid, IconButton, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Typography } from '@material-ui/core'
+import React, { useState } from 'react'
+
+import DataboomTestGraph from './DataboomTestGraph.png'
+import { IPasswordSettings } from '../Components/Profile/PasswordForm'
+import { IUser } from '../Components/Profile/ProfileForm'
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown'
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp'
+import { Link } from 'react-router-dom'
+import PasswordForm from '../Components/Profile/PasswordForm'
+import ProfileDetails from '../Components/Profile/ProfileSection/ProfileDetails'
+import ProfileForm from '../Components/Profile/ProfileForm'
+import { exampleDatasets } from '../Models/Datasets/ICompleteDatasetEntity'
+import { makeStyles } from '@material-ui/core/styles'
+
+const renderGraphRow = (row) => {
+  return (<Table size="small" aria-label="purchases">
+    <TableHead>
+      <TableRow>
+        <TableCell>Graph image</TableCell>
+        <TableCell>Datasets in graph</TableCell>
+        <TableCell>Comments</TableCell>
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      <TableRow >
+        <TableCell component="th" scope="row">
+          <img src={DataboomTestGraph} width="200" height="200" />
+        </TableCell>
+        <TableCell>{row.graphdatasets}</TableCell>
+        <TableCell>{row.comments}</TableCell>
+      </TableRow>
+    </TableBody>
+  </Table>)
+}
+
+const useRowStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset',
+    },
+  },
+})
+
+//no oxidizer, add material detials and composition, comments keep author
+function createData(id: number[], type: string, name: string, oxidizer: string, author: string, graphdatasets: string[], comments: string) {
+  return { id, type, name, oxidizer, author, graphdatasets, comments }
+}
+
+const rows = []
+
+for (let i = 0; i < exampleDatasets.length; i++) {
+  rows.push(createData([exampleDatasets[i].id], 'Dataset', exampleDatasets[i].name, exampleDatasets[i].oxidizer, exampleDatasets[i].author,
+    ['Cell width', 'Cell Height of O2 explosion', 'Critical energy after N2 intake'], 'Here is a sample comment'))
+}
+
+function createGraphData(id: number[], type: string, name: string, graphdatasets: string[], comments: string) {
+  return { id, type, name, graphdatasets, comments }
+}
+
+rows.push(createGraphData([1, 2, 3], 'Graph', 'graph name', ['Cell width', 'Cell Height of O2 explosion', 'Critical energy after N2 intake'],
+  'Here is a sample comment'))
+
+function parseIdArray(id) {
+  let stringTest = ""
+  console.log(id.length)
+  if (id.length > 1) {
+    stringTest = String(id).split(",").join("&datasetId[]=")
+    return stringTest
+  } else { return id }
+}
+
+// eslint-disable-next-line no-undef
+function Row(props: { row: ReturnType<typeof createData> }) {
```

Comment:
Does it make more sense to move this into a separate component?

Reply:
Maybe, but that section is from @laithAwad 's branch, so I didn't want to touch anything.

Reply:
After talking with @laithAwad , we decided to leave it as is. It would take more time to move it which could be used to finish up other stories, so for now the function will stay like this,